WordPress/Classes/ViewRelated/Activity/Backup/BackupListViewController+JetpackBannerViewController.swift

```diff
@@ -1,19 +0,0 @@
-import UIKit
-
-extension BackupListViewController {
-    @objc
-    static func withJPBannerForBlog(_ blog: Blog) -> UIViewController? {
```

Comment:
My understanding is that with the state of Jetpack migration, we no longer needs this. I hope we can eventually address the Jetpack migration complexity too.